docs-kits/kits/Connector Kit/Adoption View/working-with-policies.md

```diff
@@ -0,0 +1,288 @@
+                {
+                  "odrl:leftOperand": "https://w3id.org/catenax/policy/v1.0.0/UsagePurpose",
+                  "odrl:operator": {
+                    "@id": "odrl:eq"
+                  },
+                  "odrl:rightOperand": "ID 3.1 Trace"
+                }
+              ]
+            }
+          },
+          "odrl:prohibition": [],
+          "odrl:obligation": []
+        }
+      ],
+      "dcat:distribution": [
+        {
+          "@type": "dcat:Distribution",
+          "dct:format": {
+            "@id": "HttpData-PULL"
+          },
+          "dcat:accessService": "911f5da0-c9ee-4259-9a95-39428d08f767"
+        }
+      ],
+      "version": 1.0,
+      "content-type": "application/json",
+      "name": "json-1-paper",
+      "description": "Asset json-1-paper for test purposes",
+      "id": "json-1-paper"
+    }
+  ],
+  "dcat:service": {
+    "@id": "911f5da0-c9ee-4259-9a95-39428d08f767",
+    "@type": "dcat:DataService",
+    "dct:terms": "connector",
+    "dct:endpointUrl": "https://provider-dsp-end.point/api/v1/dsp"
+  },
+  "participantId": "PROVIDER-BPNL"
+}
+
+```
+
+These four Offers are of `@type` either `odrl:Offer` or one of its subclasses. More interesting however are the
+`odrl:permission`, `odrl:prohibition` and `odrl:obligation` properties. They contain `odrl:Rules` that have to be
+satisfied by the Consumer. There's two ways that can happen.
+
+### Consumer-side: `odrl:Offer` in a `ContractRequestMessage`
+
+Firstly, when transmitting a [`ContractRequestMessage`](https://docs.internationaldataspaces.org/ids-knowledgebase/v/dataspace-protocol/contract-negotiation/contract.negotiation.protocol#id-2.1-contract-request-message)
+(for example triggered via the EDC's [contract-negotiations API](https://github.com/eclipse-tractusx/tractusx-edc/blob/main/docs/usage/management-api-walkthrough/05_contractnegotiations.md)), the
+Consumer must signify consent to the Offer by including it in the Request and adding the `odrl:target` property on the
+level of an `odrl:Offer`. It is a very, very bad idea to simply copy the next-best Offer from the Catalog and replay it
+to the Provider. Instead, Consumers must perform internal checks which (if any) Offer proposes terms that they want to
+agree to. It is possible (though not usual) to encode pricing information in an `odrl:Offer`. Sending a
+`ContractRequestMessage` echoing the pricing information would represent a legally binding agreement in accordance to
+the conditions proposed by the Provider's Offer. Thus, selecting an offer extended by a Provider company should thus be
+handled with appropriate care.
+
+The Provider will usually accept the ContractRequest only if the extended Offer is semantically equivalent to his own
+as distributed via the Catalog.
+
+### Provider-side: Checking a Consumer's Verifiable Presentation
+
+In addition to checking the Offer's structure, Providers have the chance to query the Consumer's Verifiable Presentation.
```

Comment:
the connector does this! The application using the connector (EDC) does NOT perform this task. Might be a bit misleading.